css/index.css

```diff
@@ -105,25 +185,71 @@ p {
     width: 12.5%;
     height: 100px;
     background: black;
-    margin: 20px 2.5%;
+    margin: 18px 2.5%;
     color: white;
+    font-size: 2em;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Great use of some more advanced CSS :nth-child().  You could remove .middle-content from each of these if you want to.